sdk/core/azure-core/azure/core/exceptions.py

```diff
@@ -160,13 +181,14 @@ def error(self):
         return self
 
     def __str__(self):
-        return "({}) {}".format(self.code, self.message)
+        return self.message_details()
```

Comment:
Is `str()` also supposed to have a character limit? Or is that just `repr()`?

Reply:
I don't find there is a character limit in str()?

Reply:
It's in our guidelines - but looks to only be specified for `repr`:
https://azure.github.io/azure-sdk/python_implementation.html#python-models-repr-length

So str might be okay?